srcpkgs/qpdfview/template

```diff
@@ -12,8 +12,8 @@ short_desc="Tabbed document viewer"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="GPL-2.0-or-later"
 homepage="https://launchpad.net/qpdfview"
-distfiles="https://launchpad.net/qpdfview/trunk/${version}/+download/qpdfview-${version}.tar.gz"
-checksum=cc642e7fa74029373ca9b9fbc29adc4883f8b455130a78ad54746d6844a0396c
+distfiles="https://launchpad.net/qpdfview/trunk/${version}.0/+download/qpdfview-${version}.tar.gz"
```

Comment:
Why not set version as 0.5.0?

Reply:
Because full link is...

https://launchpad.net/qpdfview/trunk/0.5.0/+download/qpdfview-0.5.tar.gz

Reply:
upstream calls the release 0.5.0: https://launchpad.net/qpdfview/+announcement/30778

Reply:
As you wish...